src/Features/VisualBasic/Portable/ChangeSignature/VisualBasicChangeSignatureService.vb

```diff
@@ -429,124 +444,169 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.ChangeSignature
             Return vbnode
         End Function
 
+        Private Function IsParamsArrayExpanded(semanticModel As SemanticModel, node As SyntaxNode, symbolInfo As SymbolInfo, cancellationToken As CancellationToken) As Boolean
+            If symbolInfo.Symbol Is Nothing Then
+                Return False
+            End If
+
+            Dim argumentCount As Integer
+            Dim lastArgumentIsNamed As Boolean
+            Dim lastArgumentExpression As ExpressionSyntax
+
+            Dim invocation = TryCast(node, InvocationExpressionSyntax)
+            Dim objectCreation = TryCast(node, ObjectCreationExpressionSyntax)
+            If invocation IsNot Nothing Then
+                argumentCount = invocation.ArgumentList.Arguments.Count
+                Dim isNamed = invocation.ArgumentList.Arguments.LastOrDefault()?.IsNamed
+                lastArgumentIsNamed = isNamed.HasValue AndAlso isNamed.Value
+                lastArgumentExpression = invocation.ArgumentList.Arguments.LastOrDefault()?.GetExpression()
+            ElseIf objectCreation IsNot Nothing Then
+                argumentCount = objectCreation.ArgumentList.Arguments.Count
+                Dim isNamed = objectCreation.ArgumentList.Arguments.LastOrDefault()?.IsNamed
+                lastArgumentIsNamed = isNamed.HasValue AndAlso isNamed.Value
+                lastArgumentExpression = objectCreation.ArgumentList.Arguments.LastOrDefault()?.GetExpression()
```

Comment:
this logic looks identical to the logic above.

Reply:
It is. Deduped.